ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/CoveragesToggleGroup.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+
+import { complianceEnhancedCoveragePath } from 'routePaths';
+
+import { ComplianceProfilesContext } from './ComplianceProfilesProvider';
+
+function CoveragesToggleGroup({ tableView = 'checks' }: { tableView: string }) {
```

Comment:
Do we want to limit the types of strings we want to allow for the `tableView` variable? 

``` tableView: 'checks' | 'controls'```

Reply:
It'll be checks and clusters for now but good callout, updated.